### PR TITLE
awssqs: move to contrib

### PR DESCRIPTION
**Pull request by srvaroa**

### CODE REVIEW

**Comment by srvaroa** [comment]
Note that this fails in the current v0.3.0 (details here https://github.com/knative/docs/issues/733) but seems to work fine with the current tips of eventing and eventing-sources (https://github.com/knative/eventing/commit/5dfd2c8ee42726594d66904cd556d7799a857cda and https://github.com/knative/eventing-sources/commit/6fde0bf31f27dc744cd4fd59b7ff7f0c47b62f4c)

File: contrib/awssqs/PROJECT [hunk 1]
```diff
@@ -0,0 +1,3 @@
+version: "1"
```
Comment (matzew) [inline contrib/awssqs/PROJECT]: where is this message read ?

Comment (matzew) [reply contrib/awssqs/PROJECT]: ahm... file :-)

Comment (srvaroa) [reply contrib/awssqs/PROJECT]: I took this from gcppubsub in an attempt to keep things consistent, but I admit I don't know the purpose of the file.

Comment (srvaroa) [reply contrib/awssqs/PROJECT]: I took this from gcppubsub in an attempt to keep things consistent, but I admit I don't know the purpose of the file.

Comment (grantr) [reply contrib/awssqs/PROJECT]: It's used by kubebuilder to identify the repo root I think. I don't think it needs to be copied to contrib.

**Comment by n3wscott** [comment]
can you merge? sorry, some conflicts happened

**Comment by srvaroa** [comment]
This one has grown too stale, I didn't have time to keep it up to date in the past weeks. I will try and retake it from scratch soon.

**Pull request closed without merging**